src/test/java/com/gotocompany/depot/common/TemplateTest.java

```diff
@@ -131,4 +131,64 @@ public void shouldGetTemplatePatternFromParameterizedString() throws InvalidTemp
         Template template = new Template("http://dummy.com/%s,order_number");
         assertEquals("http://dummy.com/%s", template.getTemplateString());
     }
+
+    @Test
+    public void shouldParseCommaEnclosedInDoubleQuotes() throws InvalidTemplateException {
+        Template template = new Template("eeeee\",\"2222");
+        assertEquals("eeeee,2222", template.getTemplateString());
+    }
+
+    @Test
+    public void shouldParseMultipleCommasEnclosedInDoubleQuotes() throws InvalidTemplateException {
+        Template template = new Template("eeeee\",\"2222\",\"eeeee\",\"2222");
+        assertEquals("eeeee,2222,eeeee,2222", template.getTemplateString());
+    }
+
+    @Test
+    public void shouldParseCommaEnclosedInDoubleQuotesWithArguments() throws InvalidTemplateException {
```

Comment:
You can use the Paratmeterised test here.

Reply:
done, converted to parameterized test, thanks